doc-content/jbpm-docs/src/main/asciidoc/Integration/integration-maven-dependency-con.adoc

```diff
@@ -0,0 +1,177 @@
+[id='integration-maven-dependency-con_{context}']
+= Dependency management with Maven
+
+When you build projects that embed the {PROCESS_ENGINE}, use Apache Maven to configure all dependencies required by the {PROCESS_ENGINE}.
+
+The {PROCESS_ENGINE} provides a set of BOMs (Bill Of Material) to simplify declaring artifact dependencies.
+
+Use the top-level `pom.xml` file of your project to define dependency management for embedding the {PROCESS_ENGINE}, as shown in the following example. The example includes the main runtime dependencies, which are applicable  whether the application is deployed on an application server, in a servlet container, or as a stand-alone application. 
```

Comment:
Should be "standalone" one word, which is how you have it everywhere else anyway.